include/lib/cpus/aarch64/cortex_a53.h

```diff
@@ -41,6 +41,15 @@
 
 #define CPUECTLR_SMP_BIT		(1 << 6)
 
```

Comment:
I beg your pardon, the Advanced SIMD and Floating-point retention control bits only apply to A53, not A57.